core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.component.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for create new story modal.
+ */
+
+import { NewlyCreatedStory } from 'domain/topic/newly-created-story.model';
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { ImageLocalStorageService } from 'services/image-local-storage.service';
+import { StoryEditorStateService } from 'pages/story-editor-page/services/story-editor-state.service';
+import { TopicEditorStateService } from '../services/topic-editor-state.service';
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { ConfirmOrCancelModal } from 'components/common-layout-directives/common-elements/confirm-or-cancel-modal.component';
+import { AppConstants } from 'app.constants';
+
+@Component({
+  selector: 'oppia-create-new-story-modal',
+  templateUrl: './create-new-story-modal.component.html'
+})
+export class CreateNewStoryModalComponent extends ConfirmOrCancelModal {
+  allowedBgColors: object = AppConstants.ALLOWED_THUMBNAIL_BG_COLORS.story;
```

Comment:
Type `object` is too general to use here. You can remove the explicit type from here, the appropriate type will be auto inferred by typescript.

Reply:
Done